gulpfile.js

```diff
@@ -54,6 +59,15 @@ gulp.task('watch', function() {
     .pipe(gulp.dest(path.DEST_SRC));
 });
 
```

Comment:
please add a comment on the difference between the task 'jest' and the task 'test'